vars/parseStageInfo.groovy

```diff
@@ -177,6 +177,12 @@ def call(Map config = [:]) {
               } else {
                 // Must be a PR run
                 tag = "pr"
+
+                String target_branch = env.CHANGE_TARGET ? env.CHANGE_TARGET : env.BRANCH_NAME
+                if (target_branch == "release/1.2") {
+                  echo "Updating tag to inclue daily_regression for release/1.2"
+                  tag += " daily_regression"
+                }
```

Comment:
I wonder if this should be more generalized and driven as a _property_ of the branch's `Jenkinsfile`.  You could imagine that at any point in time that we could have a number of branches in various levels of activity and this change is really a reflection of the `release/1.2` branch's (low) activity level at the moment.  But at some point in the future, say when a 1.2.1 is being prepared the activity level could go back up and switching it back to pr with nightly runs might again become appropriate.  Other branches could be changing their activity levels at different times also.

We really shouldn't be flapping this repo to toggle such properties of a branch.

Maybe we need to define some activity levels and make generic changes here to reflect what we want happening at the different activity levels and then PRs to the branches can change that branch's activity level as needed.